packages/ui5-info/src/types.ts

```diff
@@ -46,20 +46,16 @@ export interface UI5Version {
      * UI5 semantic version
      */
     version: string;
+    support?: string;
+    lts?: string;
```

Comment:
Hi @mikicvi-SAP, do we need to change the type that is used as return type of public interface function `getUI5Versions()`?

Perhaps we can have a type that extends `UI5Version` by `support` and `lts` and is used internally. Or was this intentional to extend the return type?

Reply:
The change was intentional, as it made the most sense and it reads well. 
Of course, if you think that `UI5Version` type should be extended by another type, that can be done instead.

Reply:
Hi @mikicvi-SAP,

Consumers call `getUI5Versions()` can control the presence of properties `default` and `maintained` via options:
https://github.com/SAP/open-ux-tools/blob/13d5ee59f6ab2b7c05a74b86d78729fc1d13dad7/packages/ui5-info/src/types.ts#L30-L37

Is it also possible for a consumer to fill the newly added properties `support` and `lts`?

If not I would say we keep the public interface for return value of `getUI5Versions()` as is and create a internal interface based on it for internal handling like cache storage.

Reply:
Hi @Klaus-Keller, 

Yes, the consumer can specify the properties, calling `getUI5Versions()` will return only the version. The support is returned when the `includeMaintained` property is passed in.

As far as `lts` goes, it has been in the type before, but it hasn't been used anywhere. 
It can be added, in a similar fashion, we would have to add a new parameter `includeLts?:boolean`.

I have kept it in as I did not want to remove any properties, I don't see any usages in ui5-info so it is redundant as of now.